HelpAssistHer/help-assist-her
# Add login redirects for verification portal

## Description
This PR cleans up the login page:
- Rename "Landing" to "LoginPage" to make it more clear
- When a user successfully logs in, they are redirected to the Verification Portal form
- When a user logs out, they are redirected back to the Login page
- If a user tries to go directly to the Verification Portal form and they _are_ logged in, they'll stay there, but if they _aren't_ logged in, they'll be redirected back to the Login page
- Remove the "Verification Portal" link on the Landing page since redirects have been added
- Update the styling for the Login and Logout buttons

## Test Plan
This will have to be tested locally. We don't have things setup to login via Review Apps

- While logged out, go to http://localhost:4000/verification, you should get the login page
- Click the "Sign in with Facebook" button, you should be redirected to the form
- Click Logout, you should be redirected back to the login page
- While logged out, go to http://localhost:4000/verification/pregnancy-resource-center, you should get the Login page
- While logged _in_, go to http://localhost:4000/verification/pregnancy-resource-center, you should get the form
